packages/lazy-images/.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Files not needed to be distributed in the package.
+.gitattributes export-ignore
+phpunit.xml    export-ignore
+tests/         export-ignore
+composer.lock  export-ignore
+vendor/        export-ignore
```

Comment:
Do we need those 2 since we do not check them in?

Reply:
Yes you are right. I have now removed these entries.